scripts/run_examples

```diff
@@ -11,7 +11,7 @@ source "$SCRIPTS_DIR/common"
 # Note that the chat example cannot be run because the client is interactive.
 # TODO(#462): Re-enable chat example, even if it may be just loading the server-side code and not
 # interact with it.
-readonly RUST_EXAMPLES="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/rust$' | cut -d'/' -f2 | uniq | grep -v chat)"
+readonly RUST_EXAMPLES="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/rust$' | cut -d'/' -f2 | uniq | grep -v chat | grep -v rustfmt)"
```

Comment:
Could we get a TODO(#NNN) to re-instate the rustfmt example?

Reply:
Done